internal/command/user_human_password.go

```diff
@@ -225,9 +225,18 @@ func (c *Commands) HumanCheckPassword(ctx context.Context, orgID, userID, passwo
 		_, err = c.eventstore.PushEvents(ctx, user.NewHumanPasswordCheckSucceededEvent(ctx, userAgg, authRequestDomainToAuthRequestInfo(authRequest)))
 		return err
 	}
-	_, err = c.eventstore.PushEvents(ctx, user.NewHumanPasswordCheckFailedEvent(ctx, userAgg, authRequestDomainToAuthRequestInfo(authRequest)))
+	events := make([]eventstore.EventPusher, 0)
+	errMsg := "Errors.User.Password.Invalid"
```

Comment:
why do you create the error msg here?

Reply:
bevor I checked on the lockout policy show failure flag and if it wasnt set, I changed the err msg, but I removed this part for the moment